src/App.js

```diff
@@ -1,10 +1,12 @@
 import "./App.css";
-import {Home} from "./pages/home/Home"
+// import {Home} from "./pages/home/Home"
+import { Wishlist } from "./pages/pagesIndex";
 
 function App() {
   return (
     <div className="App">
-      <Home />
+      {/* <Home /> */}
+      <Wishlist />
```

Comment:
just a suggestion u can use the navbar and the footer component directly in the app.js if they are gonna remain the same on each of the pages .

Reply:
I will change it in last of ui